### PR TITLE
media-gfx/netgen - add tests and enable logging, drop 6.2.2105

### PR DESCRIPTION
**Pull request by waebbl**
- drop 6.2.2105
- enable logging, using dev-libs/spdlog
- enable tests, disable some tests which do not work

Signed-off-by: Bernd Waibel <waebbl-gentoo@posteo.net>

As discussed in https://github.com/gentoo/sci/pull/1180 here's a patch to look into an issue with testing the package and enable logging.

I couldn't get the python tests working. The tests seem to expect an installed version. Adding PYTHONPATH to the environment before testing didn't solve the issues. If someone has an idea on how to get this working, please share it.

### CODE REVIEW

File: media-gfx/netgen/metadata.xml [hunk 1]
```diff
@@ -5,8 +5,17 @@
 		<email>sci-mathematics@gentoo.org</email>
 		<name>Gentoo Mathematics Project</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
+		<email>waebbl-gentoo@posteo.net</email>
+		<name>Bernd Waibel</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
```
Comment (AndrewAmmerlaan) [inline media-gfx/netgen/metadata.xml:13]: Please drop proxy-maint here, they don't maintain packages in the science overlay.

Comment (waebbl) [reply media-gfx/netgen/metadata.xml:13]: Adding proxy-maint and myself was in preparation for the move to ::gentoo.
Still remove it?

Comment (AndrewAmmerlaan) [reply media-gfx/netgen/metadata.xml:13]: > Adding proxy-maint and myself was in preparation for the move to ::gentoo. Still remove it?

For adding it here we're going to have to remove it. When it is moved we can re-add it.

**Comment by AndrewAmmerlaan** [comment]
Nice work, Thanks :+1:

**Pull request closed without merging**